scripts/create-movies/create-movies.js

```diff
@@ -2,67 +2,133 @@ const axios = require("axios");
 const dotenv = require("dotenv");
 const { createReadStream, readFileSync } = require("fs-extra");
 const csv = require("csv-parser");
+const throat = require("throat");
 
 dotenv.config();
 
-(async () => {
-  const keywordsFile = readFileSync(
-    "./scripts/create-movies/Keywords.csv",
-    "utf8"
-  );
-  const [, ...lines] = keywordsFile.split("\n");
+const apiUri =
```

Comment:
I think I might've misled you with this originally. This won't ever really be run while `NODE_ENV` is 'production'. Instead of this, let's do something like: 
```apiUrl = process.env.PRODUCTION_DATABASE_URL ?? "http://localhost:8080/api/graphql"```
This allows the user running the script to specify a database URL, while falling back to their local graphql instance.

Reply:
Got it. I'll change this.